src/NUnitFramework/tests/Constraints/ExactCountConstraintTests.cs

```diff
@@ -136,5 +136,50 @@ public void FailsWhenNotUsedAgainstAnEnumerable()
             TestDelegate act = () => Assert.That(notEnumerable, new ExactCountConstraint(1));
             Assert.That(act, Throws.ArgumentException.With.Message.Contains("IEnumerable"));
         }
+
+        [Test]
+        [Description("Test that the" + nameof(ExactCountConstraint) + "return all ten elements of the given list without a triple dot")]
```

Comment:
This forms the string `Test that theExactCountConstraintreturn all…`. `return` should be `returns`, as in, "it returns."

Reply:
Also note missing space.